components/Choice/index.jsx

```diff
@@ -83,6 +85,8 @@ export default class Choice extends Component {
       requiredLabel
     } = this.props;
 
+    const optionsPerRow = this.props.optionsPerRow && parseInt(this.props.optionsPerRow, 10);
```

Comment:
Maybe it makes sense to set the optionsPerRow default to 3? So then we never need to check this.